src/System.Linq.Expressions/src/System/Dynamic/Utils/TypeExtensions.cs

```diff
@@ -44,6 +37,11 @@ private static bool MatchesArgumentTypes(this MethodInfo mi, Type[] argTypes)
             Debug.Assert(mi != null);
```

Comment:
@JonHanna thanks I get it. My "worst bad"... I didn't scroll up the page to find out another red line and the call stack... was fixated with how to access testResults.xml...